zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py

```diff
@@ -24,19 +26,27 @@ class WikipediaHandler(object):
         'description': 'Searches Wikipedia for a term and returns the top 3 articles.',
     }
 
-    def usage(self):
+    def usage(self) -> str:
         return '''
             This plugin will allow users to directly search
             Wikipedia for a specific key term and get the top 3
             articles that is returned from the search. Users
             should preface searches with "@mention-bot".
             @mention-bot <name of article>'''
 
-    def handle_message(self, message, bot_handler):
+    def handle_message(
+        self,
+        message: dict,
+        bot_handler: ExternalBotHandler
+    ) -> None:
         bot_response = self.get_bot_wiki_response(message, bot_handler)
         bot_handler.send_reply(message, bot_response)
 
-    def get_bot_wiki_response(self, message, bot_handler):
+    def get_bot_wiki_response(
+        self,
+        message: dict,
+        bot_handler: ExternalBotHandler
+    ) -> Optional[str]:
         '''This function returns the URLs of the requested topic.'''
```

Comment:
One minor concern I have is I don't think we need to break the annotation over multiple lines if they can fit on one line without sacrificing readability! Maybe take a look into seeing which one of these can be fit on one line? I think the only time we should make an annotation span over multiple lines is when there are more than half a dozen arguments or something! :)

Reply:
Hi @eeshangarg! The reason I initially spread definitions over multiple lines is if they went over 80 characters. 

```
    def get_bot_wiki_response(self, message: dict, bot_handler: ExternalBotHandler) -> Optional[str]:
```

is 101 characters (with the indent), which is why I spread it over multiple lines. Is that not a concern? If so, I'd be happy to make it go on only one line.